docs/developer-docs/latest/development/backend-customization.md

```diff
@@ -2556,10 +2555,10 @@ _Parameters:_
 
 _Parameters:_
 
-| Name   | Type   | Description                  |
-| ------ | ------ | ---------------------------- |
-| result | Object | Deleted entry                |
-| params | Object | Find params _(e.g: filters)_ |
+| Name   | Type                    | Description                                                    |
+| ------ | ----------------------- | -------------------------------------------------------------- |
+| result | Object / Array{Objects} | Deleted entry or an array of entries _(in case of bulkDelete)_ |
```

Comment:
Pretty sure out standard is doing `Object | Array` so this will need to be fixed.